openedx/openedx-platform
# feat: Export ID on Taxonomy

## Description

This adds the changes to support  the new `export_Id` field on `Taxonomy`.

## Supporting information

- Github issue: https://github.com/openedx/modular-learning/issues/183
- Internal Ticket: [FAL-3620](https://tasks.opencraft.com/browse/FAL-3620)
- openedx-learning related PR: https://github.com/openedx/openedx-learning/pull/145
- frontend-app-course-authoring related PR: https://github.com/openedx/frontend-app-course-authoring/pull/814

## Testing instructions

- Use this branch of frontend-app-course-authoring: https://github.com/openedx/frontend-app-course-authoring/pull/814
- Run `make requirements` on cms shell to install the version of openedx-learning that use `export_id`.
- Run `make migrate` on cms shell.
- Make sure you have Taxonomy/Tags data setup locally and verify that the command runs well: https://github.com/open-craft/taxonomy-sample-data/
- Verify that the tags are loaded correctly.
- Continue with the testing instructions in https://github.com/openedx/frontend-app-course-authoring/pull/814


## Other information

- [x] Before to merge bump openedx-learning version
